hyperspyui/plugins/dpc_makebivariatehistogram.py

```diff
@@ -0,0 +1,63 @@
+from hyperspyui.plugins.plugin import Plugin
+import numpy as np
+import hyperspy.api as hs
+
+
+class MakeBivariateHistogram(Plugin):
+    name = "Make bivariate histogram"
+
+    def create_actions(self):
+        self.add_action(
+                self.name + '.make_bivariate_histogram',
+                self.name,
+                self.make_bivariate_histogram,
+                tip="Make a bivariate histogram from x and y beam shift "
+                    "signals.")
+
+    def create_menu(self):
+        self.add_menuitem(
+                'DPC',
+                self.ui.actions[self.name + '.make_bivariate_histogram'])
+
+    def make_bivariate_histogram(self):
+        ui = self.ui
+        signal_wrapper_list = ui.select_x_signals(
+            2, ["Deflection X", "Deflection Y"])
+        signal0 = signal_wrapper_list[0].signal
+        signal1 = signal_wrapper_list[1].signal
+        s0_flat = signal0.data.flatten()
+        s1_flat = signal1.data.flatten()
+        spatial_std = 3
+        bins = 200
+
+        if (s0_flat.std() > s1_flat.std()):
```

Comment:
Here you should probably optimize by precalculating STD and mean, as it is used several times.